cmd/interactive/cmd.go

```diff
@@ -0,0 +1,108 @@
+package interactive
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	tea "github.com/charmbracelet/bubbletea"
+	stellarstation "github.com/infostellarinc/go-stellarstation/api/v1"
+	"github.com/infostellarinc/stellarcli/cmd/util"
+	"github.com/infostellarinc/stellarcli/pkg/apiclient"
+	"github.com/spf13/cobra"
+	"google.golang.org/protobuf/types/known/timestamppb"
+)
+
+var (
+	interactiveUse   = util.Normalize("interactive-plan [Satellite ID]")
+	interactiveShort = util.Normalize("Interactive Terminal UI.")
+	interactiveLong  = util.Normalize("Interactive Terminal UI.")
+)
+
+// Create reserve-pass command.
+func NewInteractiveCommand() *cobra.Command {
+	command := &cobra.Command{
+		Use:   interactiveUse,
+		Short: interactiveShort,
+		Long:  interactiveLong,
+		Args: func(cmd *cobra.Command, args []string) error {
+			if len(args) != 1 {
+				return fmt.Errorf("accepts 1 arg(s), received %d", len(args))
+			}
+
+			return nil
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			debugMode, _ := cmd.PersistentFlags().GetBool("debug")
```

Comment:
Should we do something with the error in case flag can't be converted to boolean?

Reply:
it's helpful for debugging, but not useful for the user of the cli.